packages/gatsby-cli/src/init-starter.js

```diff
@@ -270,7 +270,11 @@ module.exports = async (starter: string, options: InitOptions = {}) => {
 
   const hostedInfo = hostedGitInfo.fromUrl(starterPath)
 
-  trackCli(`NEW_PROJECT`, { starterName: starterPath })
+  trackCli(`NEW_PROJECT`, {
+    starterName: `${hostedInfo.domain}:${hostedInfo.user}/${
```

Comment:
we can use:
 - `hostedInfo.git()` (`git://{auth@}{domain}/{user}/{project}.git{#committish}'`) 
 - or `hostedInfo.shortcut()` (`'{type}:{user}/{project}{#committish}'`) that will normalize it

Gist for using them:
```
input: 'gatsbyjs/gatsby':
 - '.git()     ': git://github.com/gatsbyjs/gatsby.git
 - '.shortcut()': github:gatsbyjs/gatsby
input: 'https://github.com/gatsbyjs/gatsby':
 - '.git()     ': git://github.com/gatsbyjs/gatsby.git
 - '.shortcut()': github:gatsbyjs/gatsby
input: 'gatsbyjs/gatsby#v2':
 - '.git()     ': git://github.com/gatsbyjs/gatsby.git#v2
 - '.shortcut()': github:gatsbyjs/gatsby#v2
```

(produced from https://runkit.com/pieh/5d2c5e74c8552a001a2f74c4 )

Ref: https://github.com/npm/hosted-git-info/blob/latest/git-host-info.js#L50-L62

Reply:
Sweet, thanks for pointing it out. I completely missed the convenience functions from the npm readme